ambry-server/src/test/java/com/github/ambry/server/mysql/AccountStatsMySqlStoreTest.java

```diff
@@ -0,0 +1,238 @@
+/*
+ * Copyright 2020 LinkedIn Corp. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ */
+package com.github.ambry.server.mysql;
+
+import com.codahale.metrics.MetricRegistry;
+import com.github.ambry.mysql.MySqlDataAccessor;
+import com.github.ambry.mysql.MySqlMetrics;
+import com.github.ambry.mysql.MySqlUtils;
+import com.github.ambry.server.StatsHeader;
+import com.github.ambry.server.StatsSnapshot;
+import com.github.ambry.server.StatsWrapper;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.sql.Connection;
+import java.sql.Driver;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Random;
+import org.codehaus.jackson.map.ObjectMapper;
+import org.junit.Test;
+
+import static org.junit.Assert.*;
+import static org.mockito.Mockito.*;
+
+
+public class AccountStatsMySqlStoreTest {
+  private final Connection mockConnection;
+  private final MySqlDataAccessor dataAccessor;
+  private final MySqlMetrics metrics;
+  private static final String clustername = "Ambry-test";
+  private static final String hostname = "test1.ambry_1300";
+  private static final long MAX_CONTAINER_USAGE = 100000;
+  private static final long MIN_CONTAINER_USAGE = 1000;
+  private static final long DEFAULT_CONTAINER_USAGE = 5000;
+  private static final int BASE_PARTITION_ID = 100;
+  private static final int BASE_ACCOUNT_ID = 1000;
+  private static final int BASE_CONTAINER_ID = 1;
+
+  public AccountStatsMySqlStoreTest() throws SQLException {
+    mockConnection = mock(Connection.class);
```

Comment:
An integration test would be good too.  You could probably extend this class to one that uses an actual mysql db.

Reply:
integration test added